client/src/components/NavBar/AuthBars/Notifications/Notification/useStyles.ts

```diff
@@ -0,0 +1,57 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles((theme) => ({
+  notification: {
+    display: 'grid',
+    gridTemplateColumns: '25fr 75fr',
+    height: '10vh',
+    minHeight: 100,
+    width: '20vw',
+    minWidth: 500,
+    margin: '15 auto',
+    backgroundColor: 'white',
+    transitionDuration: '1s',
+    '&:hover': {
+      backgroundColor: '#9ce8cd',
+    },
+  },
+  imageContainer: {
+    width: '100%',
+    display: 'flex',
```

Comment:
Most of these styles can be applied to the Box component directly

Eg: `<Box display="flex" alignItems="center"....>